### ISSUE TITLE
[Accessibility]: add command to focus status bar

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36849]
### Audit type

Keyboard navigation

### User journey audit issue

#34059

### Problem description

On the file page ([example](https://sourcegraph.com/github.com/sourcegraph/go-diff@9d1f353a285b3094bc33bdae277a19aedabe8b71/-/blob/cmd/go-diff/go-diff.go)) with the default extension `git-extras` enabled, it is difficult to navigate to the status bar through keyboard navigation. Status bar items can be interactive; for example, `git-extras`' status bar item is a link to a commit page.

### Expected behavior

An idiom in editors (like VS Code) is exposing a command (in the keyboard navigable command palette) to focus elements that are out-of-the-way. We can add a command to our command palette to focus the first status bar item, from which users can tab to subsequent status bar items.

### Additional details

![Screen Shot 2022-06-08 at 3 40 50 PM](https://user-images.githubusercontent.com/37420160/172703198-41521588-ff4d-4ace-a354-7c60c9f5c669.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
